app/frontend/wallet/cardano-wallet.js

```diff
@@ -162,29 +162,37 @@ const CardanoWallet = async (options) => {
     return await blockchainExplorer.fetchTxInfo(txHash)
   }
 
+  function dropUnprofitableUtxos(utxos) {
+    const profitableUtxos = []
+    const witnessSize = 139
+    let addedCost = 0
```

Comment:
I would decalre the variables inputSize and addedCost inside the for loop with const instead of declaring them with "let" outside of the for loop